source/base/parameter_handler.cc

```diff
@@ -355,10 +355,10 @@ namespace
             if (const auto val = p.second.get_value_optional<std::string>())
               temp.put_value<std::string>(*val);
 
-            recursively_demangle(p.second,
-                                 temp,
-                                 is_parameter_node(p.second) ||
-                                   is_alias_node(p.second));
+            recursively_mangle_or_demangle(p.second,
+                                           temp,
+                                           is_parameter_node(p.second) ||
+                                             is_alias_node(p.second));
             tree_out.put_child(demangle(p.first), temp);
```

Comment:
So [`boost::core::demangle`](https://www.boost.org/doc/libs/1_84_0/libs/core/doc/html/core/demangle.html) works bidirectional from mangled to demangled and vice versa?